minidump-processor/src/stackwalker/arm64.rs

```diff
@@ -149,8 +149,16 @@ where
         // drowning the rest of the code in checked_add.
         return None;
     }
-    let caller_fp = stack_memory.get_memory_at_address(last_fp as u64)?;
-    let caller_lr = stack_memory.get_memory_at_address(last_fp + POINTER_WIDTH as u64)?;
+    let caller_fp = if last_fp == 0 {
```

Comment:
It would be great to have just a single `if last_fp == 0` wrapping all the memory access below for readability.